README.md

```diff
@@ -99,12 +99,21 @@ When logging in to an IdP using the Single Sign-On protocol, there are 6 steps:
 1. Service Provider grants or denies access to Principal
 
 The specification mentions that in step 4, "the principal is identified by the identity provider by some means outside the scope of \[the Single Sign-On\] profile".
-Since the authentication step is implementation specific, each IdP must write a plugin which will process the authentication of a principal.
+This means that the CTK doesn't know how to give the IdP the principal's credentials, since the authentication step is implementation specific. \
+Therefore each IdP must write a plugin which will process the authentication of a principal.
 See [4.1 Web Browser SSO Profile](https://www.oasis-open.org/committees/download.php/56782/sstc-saml-profiles-errata-2.0-wd-07.pdf) for more information on the different steps for the SSO profile.
 
+###### Examining the DDF plugin jar
+In the case of DDF's IdP plugin:
+
+1. The plugin implements the [IdpSSOResponder](external/samlconf-plugins-api/src/main/java/org/codice/compliance/saml/plugin/IdpSSOResponder.java) interface.
+1. The plugin receives the login page from DDF's IdP as an http response in the interface methods.
```

Comment:
✏️ I know that this will work fine in Markdown, but it makes me itchy for the numbers not to be sequential in the source.

Reply:
I think we should be consistent throughout the README so if you change this, you should probably change the others to increment too

Reply:
✏️ ".. as an HTTP response ..."

Reply:
I actually prefer not sequentially numbering the list in the source. That way if we need to add a step, we don't have to refactor all of the numbers.